<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearUser</class>
 <widget class="QMainWindow" name="CrearUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 210, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCrearUser">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CREAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegresar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>548</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>FECHA NACIMIENTO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCorreo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>CORREO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCorreo_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>NOMBRES:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>CONTRASEÃ‘A: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 18pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>CREAR USUARIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>APELLIDOS:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidos">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCorreoU">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContraU">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFechaN">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
